static/src/javascripts-legacy/projects/common/modules/commercial/membership-engagement-banner-parameters.js

```diff
@@ -1,69 +1,119 @@
-            return 'membership-prominent ' + selectSequentiallyFrom(['yellow', 'purple', 'bright-blue', 'dark-blue']);
-        }
+        colourStrategy: defaultColourStrategy,
+        paypalClass: 'site-message__message--show-paypal',
+        campaignCode: 'gdnwb_copts_memco_banner' // Agreed with Jess and Jesse
     };
 
-    var membershipParams = assign({}, baseParams, {
-        buttonCaption: 'Become a Supporter',
-        linkUrl: 'https://membership.theguardian.com/supporter',
-        offering: MEMBERSHIP
-    });
-
-    var contributionParams = assign({}, baseParams, {
-        buttonCaption: 'Make a Contribution',
-        linkUrl: 'https://contribute.theguardian.com/',
-        offering: CONTRIBUTIONS
-    });
-
-    var defaultParamsLookup = {
-
-        UK: assign({}, membershipParams, {
-            messageText: 'For less than the price of a coffee a week, you could help secure the Guardian’s future. Support our journalism for £5 a month.',
-            campaignCode: "mem_uk_banner"
-        }),
-
-        US: assign({}, contributionParams, {
-            messageText: 'If you use it, if you like it, then why not pay for it? It’s only fair.',
-            campaignCode: "cont_us_banner"
-        }),
-
-        AU: assign({}, membershipParams, {
-            messageText: 'We need you to help support our fearless independent journalism. Become a Guardian Australia member for just $10 a month.',
-            campaignCode: "mem_au_banner"
-        }),
-
-        INT: assign({}, membershipParams, {
-            messageText: 'For less than the price of a coffee a week, you could help secure the Guardian\'s future. Support our journalism for $7 / €5 a month.',
-            campaignCode: "mem_int_banner"
+    function engagementBannerCopy(cta) {
+        return 'Unlike many others, we haven\'t put up a paywall - we want to keep our journalism as open as we can.' + ' ' + cta
+    }
+
+    // Prices taken from https://membership.theguardian.com/<region>/supporter
+    function monthlySupporterCost(location) {
+
+        var region = geolocation.getSupporterPaymentRegion(location);
+
+        if (region === 'EU') {
+
+            // Format either 4.99 € or €4.99 depending on country
+            // See https://en.wikipedia.org/wiki/Linguistic_issues_concerning_the_euro
+            var euro = '€';
+            var amount = '4.99';
+            var euroAfterCountryCodes = [
+                'BG',
+                'HR',
+                'CZ',
+                'EE',
+                'FI',
+                'FR',
+                'DE',
+                'GR',
+                'HU',
+                'IS',
+                'IT',
+                'LV',
+                'LT',
+                'PL',
+                'PT',
+                'RO',
+                'SK',
+                'SI',
+                'ES',
+                'SE'
+            ];
+
+            return euroAfterCountryCodes.includes(location) ? amount + ' ' + euro : euro + amount;
```

Comment:
Probably yes. Let's make this a priority to fix in a future PR - perhaps @guardian/dotcom-platform should be aware. And @Calum-Campbell who added this when converted to ES6